src/components/Toolbar.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <ion-header :translucent="true">
+    <ion-toolbar>
+      <div id="title-bar" >
+        <ion-title>Product Order</ion-title>
+        <ion-title id="middle-title">{{titleText}}</ion-title>
+        <ion-button color="primary">Login</ion-button>
+      </div>
+    </ion-toolbar>
+  </ion-header>
+</template>
```

Comment:
This toolbar is shared across pages/views. I think you can transfer this under shared/components directory.